src/connection/host_lookup.c

```diff
@@ -346,74 +345,46 @@ load_hosts_file ()
       if (hostname[0] && ipaddr[0])
 	{
 	  /*not duplicated hostname, IP address or not duplicated hostname and duplicated IP address */
-	  if ((user_host_Map.find (hostname) == user_host_Map.end ()))
+	  if ((user_host_Map.find (hostname) == user_host_Map.end ())
+	      && (user_host_Map.find (ipaddr) == user_host_Map.end ()))
```

Comment:
This function can replace alias. I'd like to keep the original.

```
/etc/hosts
192.168.2.34 dev1 dev-1
```

```
cubrid_hosts.conf
192.168.2.34 dev1
192.168.2.34 dev-1
```

Reply:
I knew that etc/hosts saves first of one if there are duplicated ip address.
But in this case, this hash map function saves latest one. So, I wrote above code.

So, when you find '192.168.2.34' and dev-1 is returned.

Reply:
Ok, let's keep change.